src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -175,9 +188,15 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
     })
   }
 
+  state = {
```

Comment:
I think this is probably better at the top of the class definition (near any constructors, etc.). Even though `render` is probably a main place where you'll be referring to things.